docs/spec/lcd/api.md

```diff
@@ -0,0 +1,847 @@
+# Cosmos Hub (Gaia) LCD API
+
+This document describes the API that is exposed by the specific LCD implementation of the Cosmos
+Hub (Gaia). Those APIs are exposed by a REST server and can easily be accessed over HTTP/WS
+connections. 
+
+The complete API is comprised of the sub-APIs of different modules. The modules in the Cosmos Hub
+(Gaia) API are:
+    * ICS0 (TendermintAPI)
+    * ICS1 (KeyAPI)
+    * ICS20 (TokenAPI)
+    * ICS21 (StakingAPI)
+    * ICS22 (GovernanceAPI)
+
+
+## ICS0 - TendermintAPI
+
+The TendermintAPI describes all the endpoints that a full node exposes in order to learn about the
+status of the network.
+
+### /node_info - GET
+
+url: /node_info, Method: GET
+
+Functionality: Get LCD node status
```

Comment:
this description could be more descriptive - this doesn't tell me what to expect or why to use it